rusthelloserverless/cloud.rs

```diff
@@ -0,0 +1,26 @@
+use std::path::Path;
+
+use anyhow::Result;
+use scylla::CloudSessionBuilder;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    println!("Connecting to SNI proxy as described in cloud config yaml ...");
+
+    let session = CloudSessionBuilder::new(Path::new("config_data.yaml"))
+        .unwrap()
+        .build()
+        .await
+        .unwrap();
+
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}",
+                    &[]).await.unwrap();
```

Comment:
This should be updated to use `NetworkTopologyStrategy`, as `SimpleStrategy` is being deprecated: https://github.com/scylladb/scylladb/issues/8586